README.md

```diff
@@ -732,6 +735,21 @@ wandb_run_id:
 wandb_log_model:
 ```
 
+### Training with Deepspeed
```

Comment:
We do have a short explanation on using deep speed on line 723. perhaps these two section can be integrated or the earlier removed.